linode/region/framework_datasource_schema.go

```diff
@@ -0,0 +1,48 @@
+package region
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var frameworkDataSourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"country": schema.StringAttribute{
+			Description: "The country where this Region resides.",
+			Computed:    true,
+		},
```

Comment:
Do we want to make this as optional?

Reply:
Or the removal was intentional?

Reply:
Looks like this field being optional was a mistake in the SDKv2 implementation. I've removed it since there is no logic to resolve a region by country in this data source 👍